fire/api/firedb/base.py

```diff
@@ -63,6 +65,17 @@ def __init__(self, db=None, connectionstring=None, debug=False):
         self.debug = debug
         self._cache = new_cache()
 
+        # Konfigurer logging til fil
+        logging.basicConfig(
+            filename=pathlib.Path.home() // "fire.log",
```

Comment:
To '/'?

Reply:
Ja, det var en fejl. Jeg fjerner det her commit inden jeg merger - skulle bare have en måde at flytte koden over på en anden maskine. Vil gerne lave et ordentligt logging setup efterfølgende.